addons/api/addon/transforms/policy-object.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Transform from '@ember-data/serializer/transform';
+
+export default class PolicyObjectTransform extends Transform {
+  /**
+   * Normalize a policy object in year
+   * The API returns the policy deletion and retention information in days format,
+   * but we make the conversion to years format by rounding to the closest integer
+   * @param serialized
+   * @returns {object}
+   */
+  deserialize(serialized) {
+    if (!Object.keys(serialized)) return;
+    const years = Math.round(serialized.days / 365);
```

Comment:
Wait why do we convert these to years? Don't we want the data to stay in days since that's what gets returned from the API? If we want to display years that should be in the component

Reply:
yes, the conversion is solely for the UI. If we handle this logic in the component, we won't be needing these transform methods and I wasn't sure if we can have empty serialize/deserialize here.

Reply:
so it does seem like [transforms](https://guides.emberjs.com/release/models/defining-models/#toc_transforms) are not required?

Reply:
I agree we shouldn't be conrting to years, after I check the [storage policy CRUDL doc definition](https://docs.google.com/document/d/1yR5qkQztWnh8qTjyMXykEHEqJ0PVM3firn-8DYD38MY/edit), all the /policy/ operations expect days on requests and responses.

That said, I also see the problem of not being a primitive and being a custom type. As Zed mentions [on this comment](https://github.com/hashicorp/boundary-ui/pull/2054/files#r1434405029), could we try to use ember built in tracked here? 

If we go the custom attr route and the transform is mandatory to have, we could just return the values without any transformation. Sounds redundant tho

Reply:
So to be clear we have two options here: 
1. Use a transform and have the fields be properly tracked (the transform should never just be a passthrough since that's just like having no type in the `attr`)
2. Hoist these fields in our serializer/normalizer like we do for nested attributes and have the model just be normal strings/numbers.